src/tests/system/tests/test_authentication.py

```diff
@@ -49,7 +49,7 @@ def test_authentication__offline_login(client: Client, provider: GenericProvider
     :steps:
         1. Authenticate user with wrong password
         2. Authenticate user with correct password
-        3. Make server offline (by removing firewall rules for LDAP, KDC and Global Catalog ports)
+        3. Make server offline (by blocking communication to the provider)
```

Comment:
"traffic" sounds better than communication. It just sounds strange to me. 

s/communication/traffic 

Goes for line 70 as well.

Reply:
Done. But I can't find anything on line 70, what do you mean?

Reply:
Ah, it was in the other file. Fixed as well.